models/product-discount/src/generator.ts

```diff
@@ -0,0 +1,45 @@
+import { fake, Generator, sequence } from '@commercetools-test-data/core';
+import { TProductDiscount } from './types';
+import {
+  ClientLogging,
+  LocalizedString,
+} from '@commercetools-test-data/commons';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import * as ProductDiscountValueAbsolute from '@commercetools-test-data/product-discount-value-absolute';
+import * as ProductDiscountValueRelative from '@commercetools-test-data/product-discount-value-relative';
+import * as ProductDiscountValueExternal from '@commercetools-test-data/product-discount-value-external';
+
+// https://docs.commercetools.com/api/projects/productDiscounts#productdiscount
+
+// eslint-disable-next-line
+const [getValidFrom, _, getValidUntil] = createRelatedDates();
```

Comment:
the third element of this tuple is a "future date", which I thought would be useful for the `validUntil` property. To access it, I had to squelch the eslint error when setting the middle element to unused.

Reply:
💪